lib/Sema/CSSimplify.cpp

```diff
@@ -2896,6 +2896,30 @@ getArgumentLabels(ConstraintSystem &cs, ConstraintLocatorBuilder locator) {
   return known->second;
 }
 
+
+// Return true if the specified type conforms to the DynamicLookupMemberProtocol
+// or if it is a protocol that inherits from it.
+static bool containsOrConformsToDLMP(Type ty, DeclContext *DC,
```

Comment:
This function name doesn't read well to me. The important part is not "contains or conforms", it's "dynamic lookup member protocol". I would shorten the former and expand the latter

Reply:
Good point, switched to isDynamicMemberLookupable